light_curves/code_src/panstarrs_functions.py

```diff
@@ -62,7 +62,8 @@ def panstarrs_get_lightcurves(sample_table, *, radius=1):
         dec_column="dec_deg", 
         margin_threshold=10,
         # Optimize partition size
-        drop_empty_siblings=True
+        #drop_empty_siblings=True
+        #this parameter appears to be deprecated 2/26/25
```

Comment:
Delete unnecessary lines rather than comment them out. If needed we can always go back in the git history and check on previous versions if needed.